nautobot/docs/plugins/development.md

```diff
@@ -1124,6 +1124,9 @@ urlpatterns = [
 urlpatterns += router.urls
 ```
 
++++ 1.5.0
+    Changelog and Notes urls are now provided in the NautobotUIViewSet and NautobotUIViewSetRouter.
```

Comment:
```suggestion
    Changelog and Notes views and urls are now provided in the NautobotUIViewSet and NautobotUIViewSetRouter.
```

Reply:
The reason for this comment was to point out how you don't have to add the URLs for Notes and Changelog, then I think this was updated to also add in the NautobotUIViewSet, so now it makes more since to add what you commented.